tests/detect-dce-opnum/test.yaml

```diff
@@ -0,0 +1,24 @@
+requires:
+  min-version: 6
```

Comment:
Previous comment about this version check only applied to `dcerpc` logging so I meant this min-version check should go in the filter for `dcerpc` logs only. Here, it applies to the `alert` check too.